js/battle.js

```diff
@@ -553,7 +556,7 @@ var Pokemon = (function () {
 			if (plaintext) {
 				name += ' (' + this.species + ')';
 			} else {
-				name += ' <small>(' + this.species + ')</small>';
+				name = '<span class="pokemon" title="' +  this.species + '">' + name + ' <small>(' + this.species + ')</small>' + '</span>';
```

Comment:
That would make Ignore Opponent also ignore nicknames for the entire session wouldn't it? Since you'd keep the `.hidenicks` after each battle ends. I suppose adding it to the specific battle room (`ps-room-opaque`?) would also work right?